src/unix/udp.c

```diff
@@ -793,7 +793,8 @@ int uv__udp_try_send(uv_udp_t* handle,
   struct msghdr h;
   ssize_t size;
 
-  assert(nbufs > 0);
+  if (nbufs < 1)
```

Comment:
Any reason not to use `nbufs == 0` in these checks?

Reply:
No particular reason except that it makes the diff more Obviously Correct. `nbufs < 1` is the dual of `nbufs > 0` but `nbufs == 0` is not, not for signed types (which `nbufs` isn't but that's not immediately obvious from context)